docs/_09_development/_develop_guide/_develop_guide/_develop_new_pkg.rst

```diff
@@ -0,0 +1,320 @@
+.. _development_guide_new_pkg:
+
+------------------------
+Developing a New Package
+------------------------
+
+TDB
+
+
+
+.. _development_guide_new_pkg_architecture:
+
+Architecture of a New Package
+=============================
+
+Every new package should have the following architecture:
```

Comment:
Some changes:
- Add LICENSE and CONTRIBUTING.md
- Header files should be in /include/package_name. Also add a package_name.hpp header
- In src, a package_name.cpp and package_name_node.cpp  should appears
- Set README.md to uppercase

Separate for cpp, python and hybrid packages